rust/onnxruntime/src/lib.rs

```diff
@@ -1,20 +1,19 @@
 #![warn(missing_docs)]
 
-//! ONNX Runtime
+//! Onnxruntime
 //!
-//! This crate is a (safe) wrapper around Microsoft's [ONNX Runtime](https://github.com/microsoft/onnxruntime/)
-//! through its C API.
+//! # Build
 //!
-//! From its [GitHub page](https://github.com/microsoft/onnxruntime/):
+//! ## Environment variables
 //!
-//! > ONNX Runtime is a cross-platform, high performance ML inferencing and training accelerator.
+//! `ORT_RUST_STRATEGY`
+//! - compile Use cmake to build the Onnxruntime library (default)
+//! - system Use an installed Onnxruntime library
+//! - download Download a pre-built library
 //!
-//! The (highly) unsafe [C API](https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/session/onnxruntime_c_api.h)
-//! is wrapped using bindgen as [`onnxruntime-sys`](https://crates.io/crates/onnxruntime-sys).
+//! `ORT_RUST_LIB_LOCATION` only used if `ORT_RUST_STRATEGY=system`
```

Comment:
It would be if you compiled the Onnx Runtime yourself and had it installed in a non-standard place on your computer. I'm thinking that I should add more of an introduction to the high-level bindings. I'm going to keep the build instructions but add an introduction above.

Reply:
Sounds good!